### ISSUE TITLE
Sorting RESX display by column alphabetically

### ISSUE THREAD

**Issue by enenra** [timheuer/resx-editor#10]
Hi. Love the extension so far - it looks very nice and clean!

One thing I would love to be able to do in it is clicking on the headers (once) to sort the RESX entries A to Z by the contents of that column, and click another time to reverse the sorting.

I often find that I enter new strings haphazardly as I need them, so being able to sort them would help with my ability to keep an overview a lot.

**Comment by timheuer** [comment]
@enenra do you expect the sort to actually modify the underlying XML document itself or just a view model here?  I think this is easily attainable, but may also affect the ordering of the XML (given how I have it right now), without a bit more work.

**Comment by enenra** [comment]
No, it should probably be purely on the presentation layer, not latering the structure in the actual XML. At least that would be plenty for me. 

The one other thing that would be nice is if the editor remembered which way it was sorted last and reapplied it upon opening a resx file.

**Comment by timheuer** [comment]
I played around with this quickly this AM thinking _oh, should be simple to sort the array bound to the grid_ and yeah, that's not working ;-) -- will keep looking, but not as simple with the grid that I thought!
